cmd/uplink/cmd/setup.go

```diff
@@ -115,9 +115,27 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 			"api-key":        apiKey,
 			"enc.key":        string(encKey),
 		}
+
+		fmt.Println()
+		fmt.Println("Your Uplink CLI is configured and ready to use!")
```

Comment:
this should probably be below the actual process.SaveConfig call

Reply:
done